sql-plugin/src/main/311+-nondb/scala/com/nvidia/spark/rapids/shims/DeltaLakeUtils.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
Only current year should be mentioned for new file. 
```suggestion
 * Copyright (c) 2022, NVIDIA CORPORATION.
```

Reply:
Got it, thank you!